src/libraries/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -1764,6 +1764,7 @@ public async Task DisposedSocket_ThrowsOperationCanceledException()
         public async Task BlockingAsyncContinuations_OperationsStillCompleteSuccessfully()
         {
             if (UsesSync) return;
+            if (Environment.GetEnvironmentVariable("INLINE_SOCKET_CONTINUATIONS") == "1") return;
```

Comment:
nit: maybe prefix the environment variable with `DOTNET_` to align with others (e.g. `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT`).

Reply:
I've added the suffix. Now wait for someone to suggest a shorter name :)